### DISCUSSION TITLE
USBケーブルについて

### DISCUSSION THREAD

**Discussion by ghost** [Yowkees/keyball#341] [Q&A] [answered]
usb-c to usb-c ケーブルをPCとの接続で使いたいのですが、
usb-c to usb-a ケーブルでないと接続できません。

接続において何か条件などあるのでしょうか？

**Comment by Yowkees** [comment, marked as answer]
USB接続についてはすべてProMicroの仕様に依存します。
一部USB-C to USB-Cケーブルが使用できないProMicroがあることを伺っていまして、USBコネクタのプルダウン抵抗の有無等が関係するようです。
お手数ですがProMicroの入手先へお問合せいただければ幸いです。
